README.md

```diff
@@ -1,8 +1,7 @@
-# MPU6886-TOIT
+# MPU6886
```

Comment:
If the package.yaml doesn't have a "name", then the title of the readme is used. -> I changed it to mpu6886.